src/otx/recipe/detection/yolox_l.yaml

```diff
@@ -12,9 +12,6 @@ optimizer:
     weight_decay: 0.0001
 
 scheduler:
-  - class_path: otx.algo.schedulers.warmup_schedulers.LinearWarmupScheduler
-    init_args:
-      num_warmup_steps: 3
```

Comment:
Warmup is missing here.

Reply:
Done, https://github.com/openvinotoolkit/training_extensions/pull/3035/commits/3ccd4b1946381915bc2c179da8746a23a82f9a56